shell/common/keyboard_util.cc

```diff
@@ -36,7 +37,7 @@ ui::KeyboardCode KeyboardCodeFromKeyIdentifier(const std::string& s,
   } else if (str == "altgr") {
     return ui::VKEY_ALTGR;
   } else if (str == "plus") {
-    *shifted = true;
+    shifted_char->emplace('+');
```

Comment:
Side note: This isn't strictly true, `+` isn't a shifted character for all keyboard layouts. This PR doesn't really aim to fix that functionality, I'm just noting it here so we don't close some linked issues that do talk about that as well.

Reply:
The plus key has actually been confusing me, on Windows there seems no way to represent the <kbd>+</kbd> key, and Chromium just maps it to <kbd>Shift</kbd> + <kbd>VKEY_OEM_PLUS</kbd>.